docs/src/components/HomepageFeatures/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import clsx from 'clsx';
+import styles from './styles.module.css';
+import IdealImage from "@docusaurus/plugin-ideal-image/lib/theme/IdealImage";
+
+type FeatureItem = {
+  title: string;
+  Svg: React.ComponentType<React.ComponentProps<'svg'>>;
+  description: JSX.Element;
+};
+
+const FeatureList: FeatureItem[] = [
+  {
+    title: 'Easy to Use Object-Object Mapper',
+    Svg: require('@site/static/img/easy-to-use.svg').default,
+    description: (
+      <>
+        Mapperly is a .NET Source Generator that simplifies the implementation of object to object mappings.
+        One only needs to define the mapping methods signature. The implementation is provided by Mapperly.
+      </>
+    ),
+  },
+  {
+      title: 'Fast & Readable',
+      Svg: require('@site/static/img/fast-reliable.svg').default,
+      description: (
+          <>
+              Because Mapperly creates the mapping code at build time, there is minimal overhead at runtime.
+              Even better, the generated code is perfectly readable, allowing you to verify the generated mapping code easily.
+          </>
+      ),
+  },
+  {
+    title: 'Powered by source generators',
+    Svg: require('@site/static/img/powered-by.svg').default,
+    description: (
+      <>
+        Mapperly works by using .NET Source Generators.
+        Since no Reflection is used at runtime, the generated code is completely trimming save.
```

Comment:
I would lowercase "reflection" and change the sentence to 

> Since no reflection is used at runtime, the generated code is completely trimming save and AOT friendly.

Reply:
Fixed.